tests/fixtures/generate_test_data.py

```diff
@@ -0,0 +1,68 @@
+import json
+
+TEST_SOURCE = 'test_source'
+
+# Default return value
+# {
+#     'title': 'test record',
+#     'uuid': uuid4(),
+#     'frbr_status': 'complete',
+#     'cluster_status': False,
+#     'source': 'test_source',
+#     'authors': ['Test Author||true'],
+#     'languages': ['test_language'],
+#     'dates': ['1907|publication_date'],
+#     'publisher': ['test publisher||'],
+#     'identifiers': [],
+#     'source_id': 'test_source|test',
+#     'contributors': ['test contributor|||contributor'],
+#     'extent': '11, 164 p. ;', 
+#     'is_part_of': ['Tauchnitz edition|Vol. 4560|volume'],  
+#     'abstract': ['test abstract 1', 'test abstract 2'],
+#     'subjects': ['test_subject||'], 
+#     'rights': 'test source|public_domain|expiration of copyright term for non-US work with corporate author|Public Domain|2021-10-02 05:25:13',
+#     'has_part': [f'1|example.com/1.pdf|{test_source}|text/html|{json.dumps(flags)}'], 
+#     'flags': { 'catalog': False, 'download': False, 'reader': False, 'embed': True }
+# }
```

Comment:
I think it's safe to remove this comment! The defaults are clear in the function!